oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -189,6 +189,32 @@ reasons that we impact site operators (new translations location), we will have
 development tools as well. In addion, we will create new instructions for developers on
 how to enable translations for a new service/repo when it comes online.
 
+Final State
+***********
+
+Repositories that generate translation files will have their translation files generated
+and pushed to the openedx-translation repository via a github workflow. Once the
```

Comment:
"pushed" here means committed via a pull request?

Reply:
Yes. I'll change this so its clearer.